jobs/trigger-pynfs-on-ceph.yml

```diff
@@ -0,0 +1,49 @@
+- job:
+    name: trigger-pynfs-cephfs
+    node: cico-workspace
+    description: 'Run pynfs (NFSv4.0) tests against the latest build of Ganesha with FSAL_CEPH.'
```

Comment:
I was under the impression that would run both 4.0 & 4.2 tests... if yes, please change the description.